### DISCUSSION TITLE
coding-time/javascript/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#573] [Announcements]
# coding-time/javascript/

欢迎来到《现代JavaScript高级教程》！在这本书中，我将与您一起探索JavaScript这门令人着迷的编程语言的深度和广度。 作为一名曾在腾讯担任高级前端开发工程师的经验分享者，我深知JavaScript在当今软件开发领域中的重要性和广泛应用。它不仅仅是一门语言，更是连接着互联网世界的纽带。无论是网页应用、移动应用、服务器端开发，还是大规模的跨平...

https://linwu-hi.github.io/coding-time/javascript/

**Comment by linwu-hi** [comment]
欢迎👏🏻

**Reply by Mister-Hope** [reply]
# Read repo readme
